app/javascript/components/pages/BarShow.js

```diff
@@ -69,9 +69,8 @@ class BarShow extends Component {
                   this.props.logged_in &&
 
                   < NewCrawlModal
-                        crawls = { this.props.crawls.filter(crawl => crawl.user_id === userid) }
-                        user_id= { this.props.current_user.id }
-
+                        current_user={ this.props.current_user }
```

Comment:
Looks like current_user might not be necessary since crawls are already being filtered by current_user id here, but we can revisit this later